third-party/eazy/v1/openapi.yaml

```diff
@@ -2360,7 +2446,7 @@ components:
           description: Labels of the conversation
           type: array
           items:
-            $ref: '#/components/schemas/allLabelsResponse'
+            $ref: '#/components/schemas/labelResponse'
```

Comment:
Could not test it. Is this correct? Maybe it should be just an array of strings?